modules/unauthenticated-users-system-webhook.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+//
+// * cicd/builds/triggering-builds-build-hooks.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="unauthenticated-users-system-webhook_{context}"]
+= Adding unauthenticated users to the `system:webhook` role binding
+
+As a cluster administrator, you can add unauthenticated users to the `system:webhook` role binding in {product-title} for specific namespaces. The `system:webhook` role binding allows unauthenticated users to trigger builds from external systems such as GitHub, GitLab, and Bitbucket. Unauthenticated users do not have access to private role bindings by default. This is a change from {product-title} versions before 4.16.
+
+[IMPORTANT]
+====
+Always verify compliance with your organization's security standards when modifying unauthenticated access.
+====
+
+[NOTE]
+====
+We recommend adding using this procedure to add the `system:webhook` role binding to each specific namespace for added security.
```

Comment:
We don't say "we" in docs (per ISG [1]), and should also avoid "recommend" if possible (per SSG [2]). 

[1] https://www.ibm.com/docs/en/ibm-style?topic=word-usage#we
[2] https://redhat-documentation.github.io/supplementary-style-guide/#recommend

However, this statement is a little confusing, possibly contradictory. Isn't it actually decreasing security but allowing unauthenticated users access to this role binding?

So I don't think the "added security" part is right. 

Maybe you could just condense the statement to generically say to allow access to this role binding for each required namespace. And then say if you have a large number, you can use the cluster role binding.